_layouts/documents.html

```diff
@@ -214,8 +214,8 @@
      $(".aa-input-icon").css("display","block");
    });
 </script>
-</body>
 {% if site.google_analytics %}
 {% include google_analytics.html %}
+</body>
```

Comment:
Move the closing `body` tag behind the `if` clause

(Affects other files, too)

Reply:
https://validator.w3.org/nu/?showsource=yes&showoutline=yes&showimagereport=yes&doc=https%3A%2F%2Fdevelopers.liveperson.com%2F#l3c6

its for the Error: Stray start tag script. problem